### DISCUSSION TITLE
Chemical symbols bug

### DISCUSSION THREAD

**Discussion by Martafct** [joaomlourenco/novathesis#405] [Q&A] [answered]
Good morning. Hope you are okay. 

I just wanted to ask if there is a bug with the "chemical symbols", because it is not changing to Portuguese, like the others. 
Thank you. 

<img width="1237" alt="Captura de ecrã 2023-01-19, às 10 51 04" src="https://user-images.githubusercontent.com/115926172/213423866-6ceda607-7eff-4421-8056-4165dbda4068.png">

**Comment by joaomlourenco** [comment, marked as answer]
That's an example of how to make an "extra" (non-standard) glossary.  You name it however you want.  Did you read the comment below?
```latex
% Define a new glossary.
% \newglossary[XXg]{type_name}{XXs}{XXo}{Name to be printed}
```

That means, if you want the name in portuguese, write the name in portuguese.  :)

**Reply by Martafct** [reply]
oops, I didn't notice that before. Thanks! 

Just a final question. Regarding to Bibliography. I watched videos and I searched to try to do the bibliography. But it only appear yours reference and It adds something of mine, but it is not the bib that I want.  Am I forgetting about something? Hope you can help. If not, well I created a chapter and placed everything there. 


<img width="1269" alt="Captura de ecrã 2023-01-21, às 21 21 05" src="https://user-images.githubusercontent.com/115926172/213887656-e4ee7290-0dcc-483e-a089-a7a54532e7e1.png">
<img width="1269" alt="Captura de ecrã 2023-01-21, às 21 21 15" src="https://user-images.githubusercontent.com/115926172/213887658-8935d01b-e8ae-4830-8bc2-9fca0c5bf8a2.png">
<img width="1269" alt="Captura de ecrã 2023-01-21, às 21 21 19" src="https://user-images.githubusercontent.com/115926172/213887659-59e08120-72db-4ad5-aace-a179cf884a7d.png">
<img width="1269" alt="Captura de ecrã 2023-01-21, às 21 21 23" src="https://user-images.githubusercontent.com/115926172/213887660-89d049b8-5e80-406f-8555-cd47b746553c.png">

**Comment by joaomlourenco** [comment]
You didn't sit in your car and started driving, right?  You spent your time **learning** how to drive.   Please do the same with LaTeX!  **Learn the basics**… otherwise you'll be making silly questions and soon no one will answer them.  

I suggest you start with the “[The (Not So) Short Introduction to LaTeX2e](https://mirrors.ctan.org/info/lshort/english/lshort.pdf)” — A very comprehensive introduction to LaTeX.

Once you read it, then come back with more questions!  ;)
